src/main/kotlin/omok/model/search/Dfs.kt

```diff
@@ -0,0 +1,45 @@
+package omok.model.search
+
+import omok.model.Color
+
+abstract class Dfs(
```

Comment:
```suggestion
sealed class Dfs(
```

또한 추후에 지금보다 더 많은 Dfs 탐색 방법이 필요없을 거라 생각되는데,
sealed 로 상속을 막는 것은 어떨까 제안해봅니다.

Reply:
맞는 말씀이라고 생각됩니다!